components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -41,6 +41,7 @@ export const rewardsPanelReducer = (state: RewardsExtension.State | undefined, a
     case types.ON_WALLET_CREATED:
       state = { ...state }
       state.walletCreated = true
+      chrome.braveRewards.saveAdsSetting('adsEnabled', 'true')
```

Comment:
is this necessary? what if someone wants to create a new wallet (ex: after deleting their original one) but don't want to opt into ads?